float32 q1
float32 q2
float32 angle
float32 q1_dot
float32 q2_dot
